README.md

```diff
@@ -26,6 +26,18 @@ This project enables Java code generation in [AutoRest](https://github.com/Azure
     package com.fruit.service.spam;
     public class ApplesServiceClient { ... }
     ```
+- *--java.models-subpackage*: The sub-package that Enums, Exceptions, and Model types will be generated.
```

Comment:
The subpackage *where* Enums, Exceptions and Model types will be generated?

Reply:
Sure.